### PR TITLE
Bugfix: AppManifest as part of release assets

### PR DESCRIPTION
**Pull request by mml5bg**
# Description

This PR implements upload of AppManifest provided from CI workflow to Release assets.

Link to successful release workflow run: https://github.com/eclipse-velocitas/vehicle-app-python-template/actions/runs/4044489992

## Azure DevOps PBI/Task reference

[AB#12587](https://softwaredefinedvehicle.visualstudio.com/SoftwareDefinedVehicle/_workitems/edit/12587)

## Checklist

* [ ] Vehicle App can be started with dapr run and is connecting to vehicle data broker
* [ ] Vehicle App can process MQTT messages and call the seat service
* [ ] Vehicle App can be deployed to local K3D and is running
* [ ] Created/updated tests, if necessary. Code Coverage percentage on new code shall be >= 70%.
* [ ] Extended the documentation in Velocitas repo
* [ ] Extended the documentation in README.md

* [ ] Devcontainer can be opened successfully
* [ ] Devcontainer can be opened successfully behind a corporate proxy
* [ ] Devcontainer can be re-built successfully

* [x] Release workflow is passing

### CODE REVIEW

**Review by emsameen — CHANGES_REQUESTED** [review]
I don't think you need all this changes. 
Did you tried just to update the upload assets as:
 - name: Upload assets
        uses: softprops/action-gh-release@v1
        if: startsWith(github.ref, 'refs/tags/')
        with:
          files: |
            .vehicleApp/Documentation/release-documentation-md.zip
            ./app/AppManifest.json
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}

**Comment by emsameen** [comment]
> 

As discussed, we could keep separation of the upload assets as you have implemented now and the chances on the triggers on Pull Request will be removed after testings. 
Thanks!

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -104,6 +107,15 @@ jobs:
...
+      - name: ${{ matrix.component.Name }} -- Upload assets
+        uses: softprops/action-gh-release@v1
+        if: startsWith(github.ref, 'refs/tags/')
+        with:
+          files: |
+            ./AppManifest.json
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
       - name: ${{ matrix.component.Name }} -- Upload AppManifest.json to artifacts"
```
Comment (emsameen) [inline .github/workflows/release.yml:119]: Please remove uploading to the Artefacts, this step is not necessary anymore. That would be replaced by uploading to the Assets.

Comment (mml5bg) [reply .github/workflows/release.yml]: Done

**Review by emsameen — APPROVED** [review]
LGTM.
Please integrate only after taking the devenv workflow repo and fixing the License checker issue.
Thanks!

**Pull request merged**